lingpy/basic/wordlist.py

```diff
@@ -1044,7 +1044,35 @@ def coverage(self, stats='absolute'):
             return sum([a / self.height for a in cov.values()]) / self.width
 
     @classmethod
-    def from_cldf(cls, path, columns=[], filter=lambda row: row["Form"], *args, **kwargs):
+    def from_cldf(
+            cls, 
+            path, 
+            columns=[
```

Comment:
Isn't a mutable default argument problematic here?

Reply:
yes, I'd prefer a default, however, as this is getting VERY annoying so far. So I'll put it into a none-phrase.

Reply:
here you could simply use a tuple instead of a list.

Reply:
a, I see what you mean now. But is the new solution also fine?